enterprise/web.core/licenseinfo.xml

```diff
@@ -100,4 +100,24 @@
         <license ref="CDDL-1.1" />
         <comment type="CATEGORY_B" />
     </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/web/taglib/resources/web-jsptaglibrary_3_0.xsd</file>
+        <file>src/org/netbeans/modules/web/taglib/resources/web-jsptaglibrary_3_0.mdd</file>
+        <license ref="EPL-v20" >
+            Copyright (c) 2009, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please only include the relevant part. The license itself is already covered by the `ref` attribute, the secondary license is not important as we explicitly choose EPL. So only the copy right line (the first line here) needs to be covered addionally. this comment also applies to the other license tags.